client/src/services/lit.ts

```diff
@@ -0,0 +1,143 @@
+import { Buffer } from "buffer";
+import LitJsSdk from "lit-js-sdk";
+
+// @ts-ignore
+window.Buffer = Buffer;
+
+const client = new LitJsSdk.LitNodeClient();
+
+const ROUND_OPERATOR =
```

Comment:
assuming this is the 'hardcoded' value you were referring to to be able to test

Reply:
yeah this is the hash of the role so it should be always that one